src/Utils.js

```diff
@@ -19,7 +19,7 @@ export function setAuth(configuration, manualAuth) {
 export function assembleError({ response, error, params }) {
   if (response) {
     const customCodes = params.options && params.options.successCodes;
-    if ((customCodes || [200, 201, 202]).indexOf(response.statusCode) > -1)
+    if ((customCodes || [200, 201, 202, 204]).indexOf(response.statusCode) > -1)
```

Comment:
Is there a custom code that we'll _always_ need to pass for a certain type of operation? If so, should it be provided by default in the adaptor?

Reply:
Yes it happens that `createCase` returns a `204` so yes having it by default in the `utils.js` might be a more permanent solution